Unity-Technologies/Graphics
# Fixed LOD mesh stripping and LOD Group UI

### Purpose of this PR
This PR fixes sevevral LoD related issues.
- Fixed the LoD UI using the wrong LoD bias value (it now uses the default frame settings which is consistent with the scene view)
- LoD meshes are now correctly stripped from the build depending on the maximum LoD value contained in all HDRP assets used in the build.
- Updated the documentation with a section about LoD Management where it explains briefly the differences with builtin and stating that QualitySettings lod APIs are not supported anymore.

---
### Testing status
Did a test setup with a LOD GRoup with 4 LoDs. Each one was a different mesh asset (but not a cube/sphere otherwise they would not be stripped).
Added 3 quality levels to the project. 
- Low with specific HDRP asset containing Max lod of 2.
- Medium with no specific HDRP asset (used the default one from Graphics instead with max LoD of 1)
- High with specific HDRP asset containing max lod of 0.

In the subsequent tests I used the com.unity.build-report-inspector package (added with this url in packman) to check the stripping.
Go to window => "Open Last Build Report" after a build. Go to source assets and sort them by type to see the list of model assets contained in the build.
Did a few tests:
- Include all three quality levels: Expected result was that no mesh was stripped (because High max lod is 0).
- Remove high: Notice that lod 0 mesh is stripped.
- Remove High and Medium: Notice that lod 0 and 1 are stripped.
- Used a camera with a max lod of 1 while only having the Low settings available: Notice that lod 2 is used instead of 1 as expected.

@Unity-Technologies/gfx-qa-hdrp I did the above tests but LoD being a bit complex, I'd rather have someone else check as well that nothing was broken or that there is no remaining corner cases.
